devel/api.md

```diff
@@ -656,6 +663,46 @@ will also change the interfaces that are visible in the Nmstate state. Reducing
 the total VFs without setting `Interface.STATE` to `InterfaceState.ABSENT` for
 the corresponding interfaces might lead to an undefined system state or error.
 
+### `Ethernet.SRIOV.VFS_SUBTREE`
+
+If `Ethernet.SRIOV.TOTAL_VFS` is 1 or greater, the `Ethernet.SRIOV.VFS_SUBTREE`
+dictionary list allows to configure the VFS special options. Nmstate supports the
+following VF options:
```

Comment:
Is there maybe a good reference document that explains the options in more detail?